src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/RazorSyntaxTreeExtensions.cs

```diff
@@ -90,9 +102,7 @@ public static IReadOnlyList<CSharpStatementSyntax> GetCSharpStatements(this Razo
                 return default;
             }
 
-            var change = new SourceChange(absoluteIndex, 0, string.Empty);
-            var owner = syntaxTree.Root.LocateOwner(change);
-            return owner;
+            return GetOwner(syntaxTree, absoluteIndex);
```

Comment:
The percentage of argument validation versus actual code in this method is approaching infinity 😛